composer.json

```diff
@@ -19,7 +19,7 @@
         "php":                      ">=5.3.3",
         "symfony/framework-bundle": "~2.7|~3.0|~4.0",
         "knplabs/knp-components":   "~1.2",
-        "twig/twig":                "~1.12|~2"
+        "twig/twig":                "~1.27"
```

Comment:
even this is unexpected. Why do you want to alter this (or any other) vendor in a PR about a translation?